src/app/features/tech-record/tech-record-routing.module.ts

```diff
@@ -38,6 +39,20 @@ const routes: Routes = [
     canActivate: [MsalGuard, RoleGuard],
     resolve: { techRecord: TechRecordViewResolver }
   },
+  {
+    path: 'hide-in-vta',
+    component: TechRecordChangeVisibilityComponent,
+    data: { roles: Roles.TechRecordAmend },
+    canActivate: [MsalGuard, RoleGuard],
+    resolve: { techRecord: TechRecordViewResolver }
+  },
+  {
+    path: 'show-in-vta',
+    component: TechRecordChangeVisibilityComponent,
+    data: { roles: Roles.TechRecordAmend },
+    canActivate: [MsalGuard, RoleGuard],
+    resolve: { techRecord: TechRecordViewResolver }
+  },
```

Comment:
The ticket specifies the user should be able to do this for provisinal records as well. Seeing as the `provisional/hide-in-vta` and the `provisional/show-in-vta` routes don't exist I'm assuming it's not currently possible.

Reply:
Cheers, suggestion applied.